doc/api/buffer.markdown

```diff
@@ -393,18 +393,31 @@ byte from the original Buffer.
     // !bc
 
 
-### buf.indexOf(value[, byteOffset])
+### buf.indexOf(value[, byteOffset, encoding])
```

Comment:
This should be `buf.indexOf(value[, byteOffset[, encoding]])`. If not then the implementation is wrong.